pkg/storage/elasticsearch.go

```diff
@@ -38,6 +38,53 @@ type ElasticsearchDeployment struct {
 	Secrets    []corev1.Secret
 }
 
+func (ed *ElasticsearchDeployment) injectArguments(container *corev1.Container) {
+	container.Args = append(container.Args,
```

Comment:
Probably a matter of taste, but do you need to split this line here? The next line seems small enough to fit here.

Reply:
no I don't need to split the line not sure why I did it, or may be the make format did it for me? Anyway I will change it.